File: PHP/Triangulo.php
<?php
/**
 * @version PHP
 * @author MARIO SAM <eu@mariosam.com.br>
 * @see I would love to work with you instead solving web code tests: hire me!
 */
namespace PHP;

//echo "The result is "+calcArea(1.20, 2.30, 10.00);

class Triangulo {

    /**
     * Calculate area or perimeter.
     */
    public function calcArea($a, $b, $c) {
        $result = "";
        $maior = $c;
        $soma = $b + $a;
    
        if ( $a > $b && $a > $c ) $maior = $a;
        else if ( $b > $c ) $maior = $b;
    
        if ( $maior == $a ) $soma = $b + $c;
        else if ( $maior == $b ) $soma = $a + $c;
    
        if ( $soma > $maior ) {
            $result = "Perimetro = ". number_format(($a + $b + $c), 1);
        } else {
            $result = "Area = ". number_format(((($a + $b) * $c)/2), 1);
        }

        return $result;
    }

}

File: JAVA/Triangulo.java
/**
 * @version JAVA
 * @author MARIO SAM <eu@mariosam.com.br>
 * @see I would love to work with you instead solving web code tests: hire me!
 */
package JAVA;

import java.io.IOException;
import java.util.Scanner;

public class Triangulo {

    public static void main(String[] args) throws IOException {
		    Scanner sc   = new Scanner( System.in );
        double a = sc.nextDouble();
        double b = sc.nextDouble();
        double c = sc.nextDouble();
        sc.close();
            
        System.out.println( calcArea( a, b, c ) );
	}

    /**
     * Calculate area or perimeter.
     */
    public static String calcArea(double a, double b, double c) {
        String result = "";
        double maior = c;
        double soma = b + a;

        if ( a > b && a > c ) maior = a;
        else if ( b > c ) maior = b;

        if ( maior == a ) soma = b + c;
        else if ( maior == b ) soma = a + c;

        if ( soma > maior ) {
            result = "Perimetro = " + (a + b + c);
        } else {
            result = "Area = " + (((a + b) * c)/2);
        }

        return result;
    }
}
